### PR TITLE
New Repo: openshift-cloud-team/rebasebot 

### PR DESCRIPTION
**Pull request by Fedosin**
The bot will be used to synchronize downtream repo forks with their upstream counterparts.

Generated by `make new-repo`.

### CODE REVIEW

File: ci-operator/jobs/openshift-cloud-team/rebasebot/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvers:
+- JoelSpeed
+- elmiko
+- lobziik
+- Fedosin
+- alex-demichev
```
Comment (alexander-demichev) [inline ci-operator/jobs/openshift-cloud-team/rebasebot/OWNERS:6]:
```suggestion
- alexander-demichev
```

File: core-services/prow/02_config/openshift-cloud-team/rebasebot/OWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+approvers:
+- JoelSpeed
+- elmiko
+- lobziik
+- Fedosin
+- alex-demichev
```
Comment (alexander-demichev) [inline core-services/prow/02_config/openshift-cloud-team/rebasebot/OWNERS:6]:
```suggestion
- alexander-demichev
```

**Review by elmiko — COMMENTED** [review]
love the automation, really nice Mike!
/lgtm

**Pull request merged**